cpuLoadAgent.c:75 testTICpuLoad(vmThread, ti) failed, rc=8 (OMR_ERROR_INTERNAL)

On
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04 LTS
Release:	16.04
Codename:	xenial

```
Did
```
make -f run_configure.mk SPEC=linux_x86-64 OMRGLUE=./example/glue
make
#install numa.h
make
make test
```
Get
```
Test GetProcessCpuLoad()
callIndex: 1: rc = 13 (OMR_ERROR_RETRY), the function call is successful !
callIndex: 2: rc = 13 (OMR_ERROR_RETRY), the function call is successful !
cpuLoadAgent.c:145 rc failed, got rc=13 (OMR_ERROR_RETRY), expected rc=0 (OMR_ERROR_NONE)
cpuLoadAgent.c:149 callIndex: 3: GetProcessCpuLoad() returned a wrong error code !
cpuLoadAgent.c:75 testTICpuLoad(vmThread, ti) failed, rc=8 (OMR_ERROR_INTERNAL)
agentTest.cpp:53: Failure
Value of: rc
  Actual: 8
Expected: OMR_ERROR_NONE
Which is: 0
agent->callOnLoad() failed, OMR_ERROR_INTERNAL
```

Looks like I got further.  ~~Got this instead:~~
~~Some textSome textfvtest/omrtest.mk:57: recipe for target 'omr_porttest' failed~~
~~make[1]: *** [omr_porttest] Aborted (core dumped)~~
Actually got even further after running `make test` a second time (after enabling core dump capture.)  Killed it in the middle of ThreadExtendedTest.TestOtherThreadCputime because it was loading 100% of the system.

Relevant output for previous test:
```
[ RUN      ] CpuLoadAgentOpts/RASAgentTest.AgentC/0
cpuLoadAgent: Agent_OnLoad(options="<NULL>")
cpuLoadAgent: BindCurrentThread passed, vmThread=0x0000000001867F10
Test GetProcessCpuLoad()
NO_HISTORY
callIndex: 1: rc = 13 (OMR_ERROR_RETRY), the function call is successful !
calculateProcessCpuLoad 1 1000000 0 218221434745549
calculateProcessCpuLoad2 1 1000000 0 218221434745549
ELAPSED_TIME_NEGATIVE
callIndex: 2: rc = 13 (OMR_ERROR_RETRY), the function call is successful !
calculateProcessCpuLoad 0 1000000 218221444453040 218221434745549
callIndex: 3: rc = 0 (OMR_ERROR_NONE), the function call is successful ! Process CPU load: 0.154520
calculateProcessCpuLoad 0 1000000 218221450117748 218221444453040
callIndex: 4: rc = 0 (OMR_ERROR_NONE), the function call is successful ! Process CPU load: 0.088266
calculateProcessCpuLoad 0 1000000 218221454948771 218221450117748
callIndex: 5: rc = 0 (OMR_ERROR_NONE), the function call is successful ! Process CPU load: 0.103498
calculateProcessCpuLoad 1 1000000 0 218221454948771
calculateProcessCpuLoad2 1 1000000 0 218221450117748
ELAPSED_TIME_NEGATIVE
calculateProcessCpuLoad 1 1000000 0 218221454948771
calculateProcessCpuLoad2 1 1000000 0 218221450117748
ELAPSED_TIME_NEGATIVE
callIndex: 7: rc = 13 (OMR_ERROR_RETRY), the function call is successful !
calculateProcessCpuLoad 0 1000000 218221468683855 218221454948771
calculateProcessCpuLoad 0 1000000 218221472951804 218221468683855
calculateProcessCpuLoad 0 1000000 218221477424706 218221472951804
callIndex: 10: rc = 0 (OMR_ERROR_NONE), the function call is successful ! Process CPU load: 0.894274
callIndex: 11: rc = 10 (OMR_ERROR_NOT_AVAILABLE), the function call is successful !
callIndex: 12: rc = 10 (OMR_ERROR_NOT_AVAILABLE), the function call is successful !
callIndex: 13: rc = 11 (OMR_THREAD_NOT_ATTACHED), the function call is successful !
cpuLoadAgent: GetProcessCpuLoad() passed
Test GetSystemCpuLoad()
callIndex: 1: rc = 13 (OMR_ERROR_RETRY), the function call is successful !
callIndex: 2: rc = 13 (OMR_ERROR_RETRY), the function call is successful !
callIndex: 3: rc = 0 (OMR_ERROR_NONE), the function call is successful ! system CPU load: 0.143583
callIndex: 4: rc = 0 (OMR_ERROR_NONE), the function call is successful ! system CPU load: 0.000000
callIndex: 5: rc = 0 (OMR_ERROR_NONE), the function call is successful ! system CPU load: 0.270727
callIndex: 7: rc = 13 (OMR_ERROR_RETRY), the function call is successful !
callIndex: 10: rc = 0 (OMR_ERROR_NONE), the function call is successful ! system CPU load: 1.000000
callIndex: 11: rc = 10 (OMR_ERROR_NOT_AVAILABLE), the function call is successful !
callIndex: 12: rc = 10 (OMR_ERROR_NOT_AVAILABLE), the function call is successful !
callIndex: 13: rc = 11 (OMR_THREAD_NOT_ATTACHED), the function call is successful !
cpuLoadAgent: GetSystemCpuLoad() passed
cpuLoadAgent: UnbindCurrentThread passed
cpuLoadAgent: Agent_OnUnload
[       OK ] CpuLoadAgentOpts/RASAgentTest.AgentC/0 (113 ms)
[----------] 2 tests from CpuLoadAgentOpts/RASAgentTest (228 ms total)
```

--- a/omr/OMR_TI.cpp
+++ b/omr/OMR_TI.cpp
@@ -27,7 +27,7 @@
 #include "OMR_MethodDictionary.hpp"
 #include "OMR_VM.hpp"
 
-#define MINIMUM_CPU_LOAD_INTERVAL J9CONST_I64(10000000)
+#define MINIMUM_CPU_LOAD_INTERVAL J9CONST_I64(1000000)
 #define MAXIMUM_NEGATIVE_ELAPSED_TIME_COUNT 3
 #define UTSINTERFACE_FROM_OMRVMTHREAD(vmThread) ((vmThread)->_vm->_trcEngine? &(vmThread)->_vm->_trcEngine->omrTraceIntfS : NULL)
 

